src/components/play-pause/play-pause.tsx

```diff
@@ -92,14 +93,15 @@ class PlayPause extends Component<any, any> {
   render(props: any): VNode<any> | undefined {
     const controlButtonClass = this.props.isPlayingAdOrPlayback ? [style.controlButton, style.isPlaying].join(' ') : style.controlButton;
     const isStartOver = props.isPlaybackEnded && !this.props.adBreak;
+    const entryName = `, ${this.props.title}: ${this.props.player.sources.metadata?.name}`;
     const playbackStateText = this.props.isPlayingAdOrPlayback ? this.props.pauseText : this.props.playText;
     const labelText = isStartOver ? this.props.startOverText : playbackStateText;
     return (
       <ButtonControl name={COMPONENT_NAME}>
         <Tooltip label={labelText}>
           <Button
             tabIndex="0"
-            aria-label={labelText}
+            aria-label={labelText+ entryName}
```

Comment:
better to use `'${labelText}, ${entryName}'` and remove the `, ` from `entryName` const in line 96.

Reply:
done